server/enterprise/license.go

```diff
@@ -24,8 +24,19 @@ func NewEnterpriseChecker(api PluginAPI) Checker {
 	}
 }
 
+const (
+	e20          = "E20"
+	professional = "professional"
+	enterprise   = "enterprise"
+)
+
 func (e *enterpriseChecker) HasEnterpriseFeatures() bool {
 	config := e.api.GetConfig()
 	license := e.api.GetLicense()
-	return pluginapi.IsE10LicensedOrDevelopment(config, license)
+
+	if license != nil && (license.SkuShortName == e20 || license.SkuShortName == enterprise || license.SkuShortName == professional) {
+		return true
+	}
+
+	return pluginapi.IsE20LicensedOrDevelopment(config, license)
```

Comment:
Will this call make any different checks than the above?

Reply:
@marianunez Yes, see the code at https://github.com/mattermost/mattermost-plugin-api/blob/1aa5f947aaa680fdfcf8bfae20b2bbe17b69517e/license.go#L67-L98. It is checking for E20 features in the case that the license doesn't have a SKU present. It also checks if the system is in development mode as the final check, if the license isn't present or isn't the kind of license we're looking for.

Reply:
Nice, thanks for clarifying!